AEPCore/Sources/configuration/Configuration.swift

```diff
@@ -46,11 +46,6 @@ class Configuration: Extension {
         registerListener(type: EventType.configuration, source: EventSource.requestContent, listener: receiveConfigurationRequest(event:))
         registerListener(type: EventType.lifecycle, source: EventSource.responseContent, listener: receiveLifecycleResponse(event:))
 
-        // If we have an appId stored in persistence, kick off the configureWithAppId event
-        if let appId = appIdManager.loadAppId(), !appId.isEmpty {
```

Comment:
`loadAppId ` also loads the id from the manifest, so we couldn't simply remove it. May need to directly use `loadAppIdFromPersistence() or  loadAppIdFromManifest()` here.

Reply:
good point, fixed.